### PR TITLE
corrected saws relations

### PR DESCRIPTION
**Pull request by eu-genia**
https://github.com/BetaMasaheft/Documentation/issues/1341
https://github.com/BetaMasaheft/Documentation/issues/1307

### CODE REVIEW

File: new/LOC6818Agurba.xml [hunk 1]
```diff
@@ -64,10 +64,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </note>
                 </place>
                 <listRelation>
-                    <relation name="saws:formsPartOf"
+                    <relation name="dcterms:isPartOf"
                          active="LOC6818Agurba"
                          passive="LOC6817Zalan"/>
-                    <relation name="saws:isRelatedTo"
+                    <relation name="syriaca:has-relation-to-place"
```
Comment (thea-m) [inline new/LOC6818Agurba.xml]: in the guidelines and here http://syriaca.org/documentation/relations.html, this is listed under person-place relations

File: new/LOC6830Awesa.xml [hunk 2]
```diff
@@ -63,10 +63,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </note>
                 </place>
                 <listRelation>
-               <relation name="saws:formsPartOf"
+                    <relation name="dcterms:isPartOf"
                          active="LOC6830Awesa"
                          passive="LOC6829Gabal"/>
-                    <relation name="saws:isRelatedTo"
+                    <relation name="syriaca:has-relation-to-place"
```
Comment (thea-m) [inline new/LOC6830Awesa.xml]: same as above

File: new/LOC6818Agurba.xml (hunk 1 above)
Comment (eu-genia) [reply new/LOC6818Agurba.xml]: you are looking at an old commit

File: new/LOC6830Awesa.xml (hunk 2 above)
Comment (eu-genia) [reply new/LOC6830Awesa.xml]: same as above

Comment (eu-genia) [reply new/LOC6830Awesa.xml]: this commit was done before pull request, already corrected, other syriaca too

**Comment by thea-m** [comment]
Thank you! Have you finished working on this? Otherwise it will be confusing for me if I try to read and you go on committing...

**Comment by eu-genia** [comment]
Done now

File: 6001-7000/LOC6546Wagan.xml [hunk 3]
```diff
@@ -87,12 +87,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     </note>
                 </place>
                 <listRelation>
-                    <relation name="syriaca:has-relation-to-place"
-                         active="LOC6546Wagan"
-                         passive="LOC3291Gannat">
+                    <relation name="gn:locatedIn"
+                        active="LOC3291Gannat"
```
Comment (thea-m) [inline 6001-7000/LOC6546Wagan.xml]: I think that active always should be the present record

File: 6001-7000/LOC6554GondarineK.xml [hunk 4]
```diff
@@ -120,30 +120,30 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     </note>
                 </place>
                 <listRelation>
-                    <relation name="syriaca:has-relation-to-place"
-                         active="LOC6554GondarineK"
-                         passive="LOC3577Gondar">
+                    <relation name="gn:locatedIn"
+                        active="LOC3577Gondar"
```
Comment (thea-m) [inline 6001-7000/LOC6554GondarineK.xml]: active-passive?

Comment (eu-genia) [reply 6001-7000/LOC6554GondarineK.xml]: but then the relation does not work, I do not believe it necessarily must be moved to another record as they are harvested everywhere

Comment (thea-m) [reply 6001-7000/LOC6554GondarineK.xml]: good, let's just wait for confirmation from @PietroLiuzzo , I really want to avoid any more confusion

Comment (PietroLiuzzo) [reply 6001-7000/LOC6554GondarineK.xml]: it is ok, they are picked up from all records in the relations view, yes.

**Comment by PietroLiuzzo** [comment]
@thea-m could you please do the merging, I have not actually looked at the file, only at the conversation where I was pinged. thank you very much!

File: 6001-7000/LOC6554GondarineK.xml (hunk 4 above)
Comment (thea-m) [reply 6001-7000/LOC6554GondarineK.xml]: perfect

**Pull request merged**